sql/src/main/java/io/crate/operation/reference/sys/shard/ShardRecoveryExpression.java

```diff
@@ -37,32 +36,31 @@
     private static final String SIZE = "size";
     private static final String FILES = "files";
 
-
     public ShardRecoveryExpression(IndexShard indexShard) {
-        addChildImplementations(indexShard.recoveryState());
+        addChildImplementations(indexShard);
```

Comment:
maybe add a comment somewhere that recoveryState is mutable and that we want to get the latest state at runtime?